tooling/What-We-Use.md

```diff
@@ -0,0 +1,20 @@
+#What We Use
+
+##Commonly used tools on the team in alphabetical order
+
+(TODO: link each item to either a separate documentation file or an external page)
+
+| Name | Description | Category |
+| ---- | ----------- | ---|
+|Cisco Connect|Broad VPN|IT|
+|Github/Github desktop|Version control|Code|
+|Google calendar|Scheduling/Shared calendars|Team|
+|IntelliJ|Code editor|Code|
+|Jira|Sprint kanban board|Scrum|
+|Retrium|Retro board|Scrum|
+|Slack|Team communication|Team|
+|Vault|Key/Secret storage|Code|
+|Lodash|Js library|Code|
+|Hyperscript Helper|Js extension|Code|
+
```

Comment:
Two more additions which I'm not sure are fully team sanctioned but I have seen at least a few members use are: Insomnia and ITerm2

Reply:
@blrubenstein Yeah there is definitely an argument for merging this with the TOC. My thought process was that it might be nice to have it in the tooling directory for clarity, but we could also just make a 'tooling' section of the TOC, which now that I'm thinking of it, might be neater overall. 

We could also have different sections for 'team tools' and 'optional tools,' thoughts about that?

Reply:
Made a new commit with some updates, I moved the 'what we use' list to the readme/TOC, which I think is better, and a few other small changes. Thanks for the suggestion on that!